mm2src/coins/utxo/utxo_common.rs

```diff
@@ -826,9 +874,15 @@ where
     );
     let fut = async move {
         let fee = try_s!(coin.get_htlc_spend_fee().await);
+        let script_pubkey = match &coin.as_ref().conf.active_address_format {
```

Comment:
Encapsulation to Address struct will allow replacing this with something like `my_address.output_script()`.